README.md

```diff
@@ -66,6 +66,36 @@ module.exports = {
 }
 ```
 
+### Minimizing For Production
+
+While Webpack 5 is likely to come with a CSS minimizer built-in, with Webpack 4 you need to bring your own. To minify your the output, use a plugin like optimize-css-assets-webpack-plugin](https://github.com/NMFR/optimize-css-assets-webpack-plugin):
```

Comment:
I think you have a typo :)
"To minify ~your~ the output"

Also, I think you're missing an opening square bracket for the plugin link:
`[optimize-css-assets-webpack-plugin]`
@mike-marcacci